tools/test.py

```diff
@@ -1304,7 +1304,7 @@ def ProcessOptions(options):
   global VERBOSE
   VERBOSE = options.verbose
   options.arch = options.arch.split(',')
-  options.mode = options.mode.split(',')
+  options.mode = options.mode.lower().split(',')  # 'Release' => 'release'.
```

Comment:
I figured that it wasn't entirely obvious from context what options.mode is and why it needs to be lower-cased.  I can remove it if you feel strongly about it.